sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -129,6 +129,43 @@ VALUES ('page_validation','add_edit_event#theform_groups','/interface/main/calen
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'common#new-encounter-form', '/interface/forms/newGroupEncounter/common.php', 160, '{pc_catid:{exclusion: ["_blank"]}}', 1);
 #EndIf
 
+-- 1. users
+--
+#IfMissingColumn users weno_prov_id
+ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) NULL DEFAULT NULL;
```

Comment:
What you have is not wrong, but does not follow format of elements in database.sql.
Change it to "DEFAULT NULL"